### DISCUSSION TITLE
Problem connection on Nintendo's Online

### DISCUSSION THREAD

**Discussion by Hirils** [Raytwo/ARCropolis#523] [Issues]
Since Yesterday i struggle to play Online after using atmosphere last version 

im using 100% Wi-Fi Safe skin mod for Smash Ultimate but i cannot connect Online 

i cannot go on Nintendo Switch Service 
even Nintendo Eshop

i always have the error code : 2155-8007

**Comment by DexAe24** [comment]
Did You solve it?

**Comment by Visualwit** [comment]
PLEASE DO NOT go online with custom firmware and NSP files installed on your system. Your console will be banned. Your console may have already been flagged for a ban.
